dashboard/src/components/DeploymentForm/DeploymentErrors.tsx

```diff
@@ -55,7 +55,13 @@ class DeploymentErrors extends React.Component<IDeploymentErrorProps> {
           <NotFoundErrorAlert resource={`Application "${releaseName}"`} namespace={namespace} />
         );
       case UnprocessableEntity:
-        return <UnexpectedErrorAlert text={error && error.message} raw={true} />;
+        return (
+          <UnexpectedErrorAlert
+            text={error && error.message}
+            raw={true}
+            title="Invalid deployment"
```

Comment:
I'm concerned about calling this `deployment` because it could be confused with a kubernetes deployment kind. Maybe we should say `Invalid installation` or `No namespace selected`.

Reply:
That's a good point. `Failed installation` seems a good alternative to me.

Reply:
Good point, will change it.

Reply:
> Maybe we should say Invalid installation or No namespace selected

Couple of issues here. `Invalid installation` might not be accurate because this deploymentErrors component is shared also during upgrade. 

`No namespace selected` is too specific when this component is general purpose. AFAIK there is not an easy way to customize this title since we dispatch an error object.

In any case I think that Failed installation is good enough and better than adding the deployment confusion. Thanks!